### PR TITLE
Vtcca 3360

### PR DESCRIPTION
**Pull request by jon-pike**

### CODE REVIEW

File: app/controllers/propertyLinking/ClaimProperty.scala [hunk 1]
```diff
@@ -80,16 +80,25 @@ class ClaimProperty @Inject()(
   }
 
   def declareCapacity(uarn: Long, address: String) = authenticatedAction { implicit request =>
-    Ok(views.html.propertyLinking.declareCapacity(DeclareCapacityVM(declareCapacityForm, address, uarn)))
+    Ok(
+      views.html.propertyLinking.declareCapacity(
+        DeclareCapacityVM(declareCapacityForm, address, uarn),
+        backLink(request.headers.get("referer").getOrElse(config.newDashboardUrl("home")))))
```
Comment (darrell-plant) [inline app/controllers/propertyLinking/ClaimProperty.scala]: put all the logic in your backLink(..) helper method so you don't duplicate code

**Comment by darrell-plant** [comment]
One suggestion made 👍

File: app/controllers/propertyLinking/ClaimProperty.scala (hunk 1 above)
Comment (jon-pike) [reply app/controllers/propertyLinking/ClaimProperty.scala]: cheers Darrell - done

**Comment by darrell-plant** [comment]
LGTM 👍

**Comment by PeterPerhac** [comment]
👍 LGTPP

**Pull request merged**